torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,328 @@
+    """
+    BAND_SETS: Dict[str, Tuple[str, ...]] = {
+        "all": (
+            "VV",
+            "VH",
+            "B2",
+            "B3",
+            "B4",
+            "B5",
+            "B6",
+            "B7",
+            "B8",
+            "B8A",
+            "B11",
+            "B12",
+            "2020_Map",
+            "avg_rad"
+            ), 
+        "s1": ("VV", "VH"),
+        "s2": (
+            "B2",
+            "B3",
+            "B4",
+            "B5",
+            "B6",
+            "B7",
+            "B8",
+            "B8A",
+            "B11",
+            "B12"
+        ),
+        "esa_wc":{"2020_Map"},
+        "viirs":{"avg_rad"}
+    }
+
+    band_names = (
+        "VV",
+        "VH",
+        "B2",
+        "B3",
+        "B4",
+        "B5",
+        "B6",
+        "B7",
+        "B8",
+        "B8A",
+        "B11",
+        "B12",
+        "2020_Map",
+        "avg_rad"
+    )
+
+    rgb_bands = ["B4", "B3", "B2"]
+    modality_urls = {
+    "esa_wc": {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/esa_wc/ESA_WC.zip"},
+    "viirs": {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/viirs/VIIRS.zip"},
+    "mask": {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/mask/mask.zip"},
+    "s1" : {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s1/s1_part1.zip",
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s1/s1_part2.zip"},
+    "s2_temporal_subset" : {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_temporal_subset/s2_temporal_subset_part1.zip",
+                            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_temporal_subset/s2_temporal_subset_part2.zip"},
+    "s2_autumn" : {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part1.zip",
+                   "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part2.zip"},
+    "s2_spring" :  {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part1.zip",
+                   "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part2.zip"},
+    "s2_summer" : {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part1.zip",
+                    "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part2.zip"},
+     "s2_winter" : {"https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part1.zip",
+                    "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part2.zip"}
+    }
+    
+    split_url = "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/split_IDs/split_IDs.csv"
+    main_directory = os.path.join(os.getcwd(), "data")
+    mask_palette = {1 : (0,153,0),
+                0 : (255,255,255)} 
```

Comment:
A lot of the formatting for these is messed up. You can run `black` on these files to auto-fix most of it. See https://torchgeo.readthedocs.io/en/stable/user/contributing.html#linters for details.